book.rb

```diff
@@ -0,0 +1,11 @@
+require './rental'
+
+class Book
```

Comment:
- [ ] Your Book class is missing the other end of the has many through association. Kindly add a method like so:

  ```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
  ```

Reply:
Thank you for the feedback. I will implement all changes.